.travis.yml

```diff
@@ -20,15 +29,15 @@ addons:
       - gcc
       - binutils-dev
       - protobuf-compiler
+      - librocksdb-dev
 
 env:
   global:
+    - ROCKSDB_LIB_DIR=/usr/lib/ # Use dynamically linked RocksDB library to speed up builds
```

Comment:
Does this mean that end users will have to install rocksdb in order to run our binaries?

Reply:
Not at all! Users can opt-in for dynamic linking of RocksDB by setting this environment variable, but the default is still local compilation of the C library + static linking. This line is only for Travis.

Reply:
I see, nice!